packages/core/src/ctype/CType.ts

```diff
@@ -96,24 +95,28 @@ export function getIdForSchema(
  * @param object Data to be verified against schema.
  * @param schema Schema to verify against.
  * @param messages Optional empty array. If passed, this receives all verification errors.
- * @returns Whether or not verification was successful.
  */
 export function verifyObjectAgainstSchema(
   object: Record<string, any>,
   schema: Record<string, any>,
   messages?: string[]
-): boolean {
+): void {
   const validator = new JsonSchema.Validator(schema, '7', false)
   if (schema.$id !== CTypeModel.$id) {
     validator.addSchema(CTypeModel)
   }
   const result = validator.validate(object)
-  if (!result.valid && messages) {
-    result.errors.forEach((error) => {
-      messages.push(error.error)
-    })
+  if (!result.valid) {
```

Comment:
Do you think destructuring and an early return will make it more elegant?

Reply:
that would work, yes. why not.